DTS-STN/secure-client-hub
# 119814 modal displays 0

## [ADO-119814](https://dev.azure.com/VP-BD/DECD/_workitems/edit/119814)

Fixed [AB#119814](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/119814)

### Description

List of proposed changes:

- Ensured that modal will not pop up unless the user has been inactive for a minimum period of time. 
- Inactivity is defined as mouse movements (PC) and scrolls (Mobile device)

### What to test for/How to test

 - Go to my-dashboard and jiggle the mouse for one minute. Nothing should happen. 
 - Leave idle for 30 seconds, timer modal should appear
 - Switch to mobile interface, scroll up and down for one minute. Nothing should happen. 
 - Leave idle on mobile interface for 1 minute. Modal should appear. 
 - When modal appears, click Signout button. User should be redirected. 
 - When modal appears, click on "Stay signed in" and repeat previous steps. Modal should act as expected. 

### Additional Notes
